projects/otsea/index.js

```diff
@@ -0,0 +1,68 @@
+
+const { GraphQLClient, gql } = require("graphql-request");
+const ADDRESSES = require('../helper/coreAssets.json')
+const { staking } = require("../helper/staking");
+const config = require("./config");
+
+const OTSEA_TOKEN = "0x5dA151B95657e788076D04d56234Bd93e409CB09";
+const OTSEA_MARKET_CONTRACT = "0x6E8B67B315b44519f8C2BEfdbbE11097c45353b4";
+const OTSEA_STAKING_CONTRACT = "0xF2c8e860ca12Cde3F3195423eCf54427A4f30916";
+
+// OTSEA market volume by network
+function getMarketVolume(endpoint) {
+    return async (api) => {
+        // locked in sell orders (ETH amount on sell orders)
+        let graphQLClient = new GraphQLClient(endpoint);
+        let query = gql`
```

Comment:
Where is the ETH in the sell orders held?

Reply:
We have buy/sell orders on markets.  
Buy orders means the request of buying the target token with ETH, so users lock ETH. we calculate locked ETH for buy orders by fetching the ETH balance of our market contract.
Sell orders means the request of selling the token. so users lock the target token and suggest the required ETH amount. We calculate the sum of required ETH amount for sell orders by using our graphql endpoints.

Totally, locked assets = locked ETH (buy orders) + required ETH (sell orders)

Reply:
is it possible to count the token opposed to the required ETH? If not, could you add a 'misrepresentedTokens: true' property to your adapter exports?

Reply:
Added 'misrepresentedTokens: true' property to exports